docker_daemon/check.py

```diff
@@ -739,6 +742,25 @@ def _format_events(self, aggregated_events, containers_by_id):
 
         return events
 
+    def _report_exit_codes(self, events, containers_by_id):
+        for event in events:
+            container_tags = set()
+            container = containers_by_id.get(event.get('id'))
+            # Skip events related to filtered containers
+            if container is not None and self._is_container_excluded(container):
+                continue
+
+            # Report the exit code in case of a DIE event
+            if container is not None and event['status'] == 'die':
```

Comment:
Reporting non-failing exits could be useful, to compute the failing/total ratio. but that might be overkill